pkgs/applications/networking/mailreaders/neomutt/default.nix

```diff
@@ -19,6 +19,8 @@ stdenv.mkDerivation rec {
   patches = [
     # https://github.com/neomutt/neomutt/issues/3773#issuecomment-1493295144
     ./fix-open-very-large-mailbox.patch
+    # Cherry-pick https://github.com/neomutt/neomutt/pull/3933
+    ./disable-faulty-char-tests.patch
```

Comment:
Since this is already merged, I would rather go with `fetchpatch` and the url https://github.com/neomutt/neomutt/pull/3933.patch instead of commiting the file.

Reply:
Ah, I thought this was neater because the patch doesn't cleanly apply (specifically the `.github/workflows/macos.yml` hunk fails to apply)

Reply:
Oh ok, then its fine.

Reply:
fwiw `fetchpatch` accepts an `excludes` argument which is precisely for cases like this, i.e. `excludes = [ ".github/workflows/macos.yml" ];`.

Reply:
💡